react/features/base/participants/middleware.js

```diff
@@ -34,7 +46,18 @@ declare var APP: Object;
 MiddlewareRegistry.register(store => next => action => {
     const { conference } = store.getState()['features/base/conference'];
 
+    if (action.type === PARTICIPANT_JOINED
+        || action.type === PARTICIPANT_LEFT) {
+        _maybePlaySounds(store, action);
```

Comment:
The switch case for PARTICIPANT_JOINED is mixed with PARTICIPANT_UPDATED. I didn't know how to deal with that.

Reply:
The case is actually not combined, they're just after each other and the JOINED case has no break statement, so the flow just continues with the UPDATED case. You should be safe adding a new line just after the JOINED case and not adding a break - that should behave as you expect, but better test it :)

Reply:
hmm maybe I'd rather not to do that given that one has to stop and think if it will work as expected. It will probably generate eslint error about switch fallthrough. Playing sound is kind of separate from other participant doings, so maybe lets keep that in separate function ?

Reply:
Yeah, maybe you're right about eslint and the logic, ok